### DISCUSSION TITLE
Error running version 2.19

### DISCUSSION THREAD

**Discussion by neilsardesai** [peripheryapp/periphery#744] [General]
Hello,
Just updated to Periphery 2.19, but now the same command that worked in 2.18 is producing an error:

Command:

`periphery scan --skip-build --index-store-path '[REDACTED]' --project '[REDACTED]' --schemes '[REDACTED]' --targets '[REDACTED]'
`

Error:

`* Inspecting project...
error: Shell command '/bin/sh -c xcodebuild -project '[REDACTED]' -list -json' returned exit status '65’:`

Anyone else running into this?

**Comment by neilsardesai** [comment]
Maybe it might be related to having a space in the project name? Tried a different project with no space in the name and it worked fine there 🤔

**Comment by neilsardesai** [comment]
Was able to work around this with `--skip-schemes-validation`

**Reply by ileitch** [reply]
There was a change in 2.19.0 related to the `xcodebuild -list` command, however, the output you pasted indicates it should be the same as 2.18.0. Does `xcodebuild -project '[REDACTED]' -list -json` succeed when you run it directly from the command line?

**Reply by neilsardesai** [reply]
Thanks for getting back to me, Ian. `xcodebuild -project '[REDACTED]' -list -json` does succeed. I was able to confirm that the issue occurs when the project name has a space in it.

I created two new projects, one called `New Project.xcodeproj` and one called `NewProject.xcodeproj`. I built both projects successfully with `xcodebuild`. 

However 

`periphery scan --skip-build --index-store-path '../dd/Index.noindex/DataStore/' --project 'New Project.xcodeproj' --schemes 'New Project' --targets 'New Project'`

fails with `error: Shell command '/bin/sh -c xcodebuild -project /Users/neil/Desktop/New Project/New Project.xcodeproj -list -json' returned exit status '65'`

whereas 

`periphery scan --skip-build --index-store-path '../dd/Index.noindex/DataStore/' --project 'NewProject.xcodeproj' --schemes 'NewProject' --targets 'NewProject'`

succeeds

**Reply by ileitch** [reply]
Thanks for taking a look. I've opened https://github.com/peripheryapp/periphery/issues/747 to investigate further.
